src/views/dmca/dmca.scss

```diff
@@ -0,0 +1,2 @@
+@import "../../colors";
+@import "../../typography";
```

Comment:
Although that is a good idea, I'm still leaning towards keeping the `dmca.scss` file in case changes are needed in the future.